### PR TITLE
Add GpgFrontend.app v2.0.10

### PR DESCRIPTION
**Pull request by saturneric**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/gpgfrontend.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask "gpgfrontend" do
+  version "2.0.10"
+  sha256 "4682eabb518d2211200f7ba0440cbb1de60ee6c513e995b14f93cf343fa35214"
+
+  url "https://github.com/saturneric/GpgFrontend/releases/download/v#{version}/GpgFrontend-#{version}-macos-11-x86_64.dmg", verified: "github.com/saturneric/GpgFrontend/"
```
Comment (SMillerDev) [inline Casks/gpgfrontend.rb:5]: No arm builds?

File: Casks/gpgfrontend.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  name "GpgFrontend"
+  desc "Powerful, Easy-to-Use, Compact OpenPGP(pgp)/GnuPG(gpg) Crypto Tool"
+  homepage "https://gpgfrontend.pub/"
+
+  depends_on macos: ">= :big_sur"
+  depends_on formula: "gnupg"
+
+  app "GpgFrontend.app"
+end
```
Comment (SMillerDev) [inline Casks/gpgfrontend.rb:14]: Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/gpgfrontend.rb (hunk 1 above)
Comment (saturneric) [reply Casks/gpgfrontend.rb:5]: It can run on arm. Tested.

Comment (SMillerDev) [reply Casks/gpgfrontend.rb:5]: But needs Rosetta?

Comment (saturneric) [reply Casks/gpgfrontend.rb:5]: I just drag into "Application" and double-click. It runs smoothly without any extra steps.

File: Casks/gpgfrontend.rb (hunk 2 above)
Comment (saturneric) [reply Casks/gpgfrontend.rb:24]: done

File: Casks/gpgfrontend.rb (hunk 1 above)
Comment (saturneric) [reply Casks/gpgfrontend.rb:5]: I test it using two different MacBooks in an Apple Store.

Comment (saturneric) [reply Casks/gpgfrontend.rb:5]: It builds in an x86 machine with macOS 11 installed using Xcode.

Comment (saturneric) [reply Casks/gpgfrontend.rb:5]: Yeah, the whole project is cross-platform and universal.

File: Casks/gpgfrontend.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+cask "gpgfrontend" do
+  version "2.0.10"
+  sha256 "4682eabb518d2211200f7ba0440cbb1de60ee6c513e995b14f93cf343fa35214"
+
+  url "https://github.com/saturneric/GpgFrontend/releases/download/v#{version}/GpgFrontend-#{version}-macos-11-x86_64.dmg", verified: "github.com/saturneric/GpgFrontend/"
+  name "GpgFrontend"
+  desc "Powerful, Easy-to-Use, Compact OpenPGP(pgp)/GnuPG(gpg) Crypto Tool"
```
Comment (bevanjkay) [inline Casks/gpgfrontend.rb:7]: @saturneric Can you please rewrite the `desc` stanza to conform to the guidelines - https://docs.brew.sh/Cask-Cookbook#stanza-desc

**Review by bevanjkay — DISMISSED** [review]

**Comment by miccal** [comment]
What are the differences between `GpgFrontend-2.0.10-macos-10.15-x86_64.dmg`, `GpgFrontend-2.0.10-macos-11-x86_64.dmg` and `GpgFrontend-2.0.10-macos-12-x86_64.dmg`, and why have you chosen to use the `GpgFrontend-2.0.10-macos-11-x86_64.dmg` one?

File: Casks/gpgfrontend.rb (hunk 3 above)
Comment (saturneric) [reply Casks/gpgfrontend.rb]: Done @bevanjkay .

**Comment by saturneric** [comment]
> What are the differences between `GpgFrontend-2.0.10-macos-10.15-x86_64.dmg`, `GpgFrontend-2.0.10-macos-11-x86_64.dmg` and `GpgFrontend-2.0.10-macos-12-x86_64.dmg`, and why have you chosen to use the `GpgFrontend-2.0.10-macos-11-x86_64.dmg` one?

They build on different version of macOS system. Users using newer macOS operating system may choose to use `GpgFrontend-2.0.10-macos-12-x86_64.dmg`. Support for macOS 10.15 still provide, but not recommend anymore. It will be disappeared from release soon. The major support is for macOS 11, with a better compatibility for most users.  So, I choose this artifact to provide.

**Comment by miccal** [comment]
Thank you @saturneric.

**Review by miccal — APPROVED** [review]

**Comment by saturneric** [comment]
Thanks all.

**Pull request merged**